website_cms/templates/menu.xml

```diff
@@ -42,7 +41,28 @@
 
 </template>
 
-</data>
-</openerp>
+<template id="submenu" name="Website CMS Submenu">
```

Comment:
@PCatinean any way to make this readable? :)

Maybe we can move this logic to a `website` helper method?